utils/geom_2d/lin_alg.py

```diff
@@ -0,0 +1,159 @@
+# This file is part of project Sverchok. It's copyrighted by the contributors
+# recorded in the version control history of the file, available from
+# its original location https://github.com/nortikin/sverchok/commit/master
+#
+# SPDX-License-Identifier: GPL3
+# License-Filename: LICENSE
+
+"""
+This module is for geometry functions handle meshes in 2D space ("XY" surface) mostly.
+"""
+
+
+x, y, z = 0, 1, 2
+
```

Comment:
I'd suggest to apply either `numpy` or `mathutils.geometry` for this module.

Reply:
I would like to avoid Blender modules. Using numpy is probably good idea but I'm not ready yet.